assets/base.css

```diff
@@ -221,6 +221,13 @@ html.no-js .no-js-hidden {
   margin: 0 auto;
 }
 
+@media screen and (max-width: 749px) {
+  .page-width-mobile {
```

Comment:
The thing with `page-width` is that it has styling for each breakpoint but I wanted something that only applies to mobile and not any other breakpoint. To avoid having to overwrite it. 
But there might be a better approach to it 🤔 
Ill take a look tomorrow and see how else I could make it work.